src/compiler/core.ts

```diff
@@ -1829,7 +1829,9 @@ namespace ts {
         return i < 0 ? path : path.substring(i + 1);
     }
 
-    export function combinePaths(path1: string, path2: string) {
+    export function combinePaths(path1: Path, path2: string): Path;

```

Comment:
This is not correct.. Path handle case sensitivity.. since path2 doesnt have Path as type you would never be sure if combined paths would be Path or just string (that needs to handle case sensitivity)